docs-markdown/src/test/suite/controllers/image-controller.test.ts

```diff
@@ -44,17 +44,7 @@ const context: ExtensionContext = {
 	storagePath: '',
 	globalStoragePath: '',
 	logPath: '',
-	extensionUri: Uri.parse(uri),
-	environmentVariableCollection: {
```

Comment:
Fixes

<img width="1470" alt="Screen Shot 2020-06-29 at 6 37 00 PM" src="https://user-images.githubusercontent.com/46545485/86074265-71e62880-ba3a-11ea-86a7-20892fe18ba0.png">

Reply:
This error actually was related to me having a newer version of the `@types/vscode` package because my package-lock.json had a different version in it then what was in package.json.  

In this repository I see 8 extensions, 6 of which commit their package-lock.json.  Only the *docs-images* and *docs-markdown* extensions do **not** commit their package-lock.json.

per the documentation [configuring-npm](https://docs.npmjs.com/configuring-npm/package-lock-json.html)

`This file is intended to be committed into source repositories, and serves various purposes`

I have added a new commit to this PR to commit this file.  I'm not sure why this was originally .gitignored, but please review this comment.

Reply:
disregard will submit in separate PR